src/components/scrim/edit-scrim-form.tsx

```diff
@@ -81,6 +91,7 @@ export function EditScrimForm({
     defaultValues: {
       name: scrim.name ?? "",
       teamId: (scrim.teamId ?? 0).toString(),
+      date: scrim.date ? new Date(scrim.date) : new Date(),
```

Comment:
This should just be `scrim.date`, it is a property that should be on every scrim
![CleanShot 2024-06-12 at 14 45 17@2x](https://github.com/luxdotdev/parsertime/assets/55987186/21a3aa5a-0423-4dca-9a10-ea66ce1191ea)
The typing here should tell you that this cannot be null or undefined